packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/clusters/nodes/node/node.routing.js

```diff
@@ -20,10 +20,13 @@ export default /* @ngInject */ ($stateProvider) => {
         DedicatedServerInterfacesService,
         specifications,
       ) =>
-        DedicatedServerInterfacesService.getInterfaces(
-          serverName,
-          specifications,
-        ),
+        specifications.constructor === Object &&
+        Object.keys(specifications).length !== 0
```

Comment:
What is the value of specifications which forces that check ?
Moreover, you should move the check directly in the service to avoid duplication your code if the service is called in different places

Reply:
the value is {} created by this line https://github.com/ovh/manager/blob/master/packages/manager/modules/bm-server-components/src/server/server.service.js#L1109 ok I move it in the service